app/src/main/java/info/nightscout/androidaps/dialogs/WizardDialog.kt

```diff
@@ -241,6 +246,26 @@ class WizardDialog : DaggerDialogFragment() {
             }, fabricPrivacy::logException)
         )
 
+        setA11yLabels()
+    }
+
+    private fun setA11yLabels() {
+        val bgInputTextId = binding.bgInput.editText?.id
```

Comment:
kotlin way is:
`binding.bgInput.editText?.id?.let { binding.bgInputLabel.labelFor = it }`

Reply:
Fixed. Thanks for the feedback, I like to learn.